addons/web/static/src/views/fields/attachment_image/attachment_image_field.xml

```diff
@@ -3,7 +3,10 @@
 
     <t t-name="web.AttachmentImageField">
         <div class="o_attachment_image">
-            <img t-if="props.record.data[props.name]" t-attf-src="/web/image/{{ props.record.data[props.name][0] }}?unique=1" t-att-title="!!env.debug and props.record.data[props.name][1]" alt="Image" />
+            <img t-if="props.record.data[props.name]"
+                t-attf-src="/web/image/{{ props.record.data[props.name][0] }}/250x250?unique=1"
```

Comment:
Why not resizing the image to have a picture with 300x300 as size? 

By doing that, I think we will not need the CSS style you added in kanban_controller.scss

What do you think? 🙂

Reply:
Indeed, I was scared about big resolutions (ex: 4K) but 300x300 is ok for all resolutions that I tried, thanks :+1: Updated